recompile.sh

```
@@ -5,10 +5,14 @@ set -euo pipefail
 VCPKG_PATH=$VCPKG_PATH/vcpkg/scripts/buildsystems/vcpkg.cmake
-BUILD_TYPE=${2:-"linux-release"}
-EXTRA_CMAKE_ARGS=("${@:3}")
 ARCHITECTURE=$(uname -m)
-ARCHITECTUREVALUE=0
+DEFAULT_BUILD_TYPE="linux-release"
+if [[ "$ARCHITECTURE" == "aarch64"* ]]; then
+	DEFAULT_BUILD_TYPE="linux-release-arm"
+fi
+BUILD_TYPE=${2:-"$DEFAULT_BUILD_TYPE"}
+EXTRA_CMAKE_ARGS=("${@:3}")
+IS_ARM64=0
```

Shellcheck (0.11.0)

[warning] IS_ARM64 appears unused. Verify use (or export if used externally).

(SC2034)